tests/integration/pilot/common/routing.go

```diff
@@ -3829,8 +3829,8 @@ spec:
 			statusCode: http.StatusOK,
 			from:       t.Apps.A,
 			to:         fakeExternalAddress,
-			protocol:   protocol.HTTPS,
-			port:       443,
```

Comment:
can you exxplain why this changed

Reply:
the test before was expecting to send to a  bogus IP with an SNI set. With auto allocation there is a VIP and Istio will configure a listener for an IP match, not an SNI match